hztb-rest-api/src/main/java/com/svs/hztb/api/sm/model/group/GroupOutput.java

```diff
@@ -11,8 +11,9 @@
 	    private Status status;
 		private ErrorOutput errorOutput;
 		private List<GroupDetail> groupDetailList = new ArrayList<GroupDetail>();
-		
 		public Status getStatus() {
+			
+			int i=1;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "i" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-52-23-152-124.compute-1.amazonaws.com:9000/coding_rules#rule_key=squid%3AS1481)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "i". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-52-23-152-124.compute-1.amazonaws.com:9000/coding_rules#rule_key=squid%3AS1854)